src/osyris/io/sink.py

```diff
@@ -45,7 +45,7 @@ def initialize(self, meta, select):
             m = meta['unit_d'] * meta['unit_l']**3 * units.g  # noqa: F841
             l = meta['unit_l'] * units.cm  # noqa: F841, E741
             t = meta['unit_t'] * units.s  # noqa: F841
-            if u == '1':
+            if u == '1' or u == '1 ':
```

Comment:
Would
```suggestion
            if u.strip() == '1':
```
be a more robust way of performing this check?

Reply:
Yes I agree with your suggestion